aws_cloudtrail_rules/aws_s3_bucket_deleted.yml

```diff
@@ -11,6 +11,12 @@ Severity: Info
 Description: A S3 Bucket, Policy, or Website was deleted
 Runbook: Explore if this bucket deletion was potentially destructive
 Reference: https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteBucket.html
+SummaryAttributes:
+  - awsRegion
```

Comment:
Would `eventName` be useful here?

Reply:
No, the `.py` file explicitly checks for `event['eventName'].startswith('DeleteBucket')` so this should only ever yield `eventName` := `DeleteBucket`